src/Connection.php

```diff
@@ -163,7 +163,6 @@ class Connection
      * @param Configuration|null  $config       The configuration, optional.
      * @param EventManager|null   $eventManager The event manager, optional.
      * @psalm-param Params $params
-     * @phpstan-param array<string,mixed> $params
```

Comment:
PHPStan fully understands our `Params` type now, so we can have it analyze the `@psalm-param` annotation.